src/lib/tuning/tuning_operation.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <iostream>
+
+namespace opossum {
+
+/**
+ * A TuningOperation is derived from a TuningChoice and either
```

Comment:
“Derived” has a specific meaning when we are in the world of classes

Reply:
done